utils/python/CIME/build.py

```diff
@@ -411,36 +411,35 @@ def build_checks(case, build_threaded, comp_interface, use_esmf_lib, debug, comp
     case.set_value("BUILD_COMPLETE", False)
 
     case.flush()
+    return sharedpath
```

Comment:
It seems unintuitive to have build_checks return the sharedpath. At the very least, there should be some function-level documentation of this return value.